src/style/animatableManager.js

```diff
@@ -0,0 +1,122 @@
+import _ from 'lodash';
+
+
+/** Definitions */
+const definitions = {
+  basicListEntrance: {
+    from: {opacity: 0, translateY: 20},
+    to: {opacity: 1, translateY: 0},
+  }, 
+  cardEntrance: {
+    from: {opacity: 0, translateY: 40},
+    to: {opacity: 1, translateY: 0},
+  },
+};
+
+class AnimatableManager {
+  loadCustomDefinitions(costumDefinitions) {
+    return {
+      ...definitions,
+      ...costumDefinitions,
+    };
+  }
+
+  /** Presets */
+  // Commons
+  getFadeIn() {
+    return {
+      animation: 'fadeIn',
+      duration: 300,
+      easing: 'ease-out-quint',
+      useNativeDriver: true,
+    };
+  }
+
+  // Components
+  getDialogAnimationProps(options) {
+    return {
+      animation: 'slideInUp',
+      duration: 400,
+      useNativeDriver: true,
+      ...options,
+    };
+  }
+
+  getCardEntranceAnimationProps(index) {
```

Comment:
This preset also comes from the example screens, if we do want to include it as part of presets it has to have more clear name, for instance, this one creates a random appearance (with a random delay), I would make it way more generic so that you can send a name of an animation, and it will return a preset with that name with the random delay behavior. 
also, currently there's a bug here, if the `index` was not sent, I think you'll get an error. 

In general, I think the methods should behave more as tools and less than just tons of presets

Reply:
Done.